{"name": "困人处置列表-真实", "status": "passed", "attachments": [{"name": "log", "source": "c3698500-e715-4436-bc65-6347c0d252d2-attachment.txt", "type": "text/plain"}], "start": 1692598007238, "stop": 1692598007493, "uuid": "8f854c9a-8cf7-40f8-8e8b-e69c6831d771", "historyId": "2c9991b785716977eb4cbbaa1f323277", "testCaseId": "792deb5b60e48cc51a654ad1e9657ccb", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_003", "labels": [{"name": "story", "value": "应急处置"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "15816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}